.github/workflows/ci.yml

```diff
@@ -343,6 +347,8 @@ jobs:
           NEXT_API_URL: ${{ secrets.CODECOV_API_URL }}
           NUXT_UPLOAD_TOKEN: ${{ secrets.CODECOV_ORG_TOKEN }}
           NUXT_API_URL: ${{ secrets.CODECOV_API_URL }}
+          OIDC_UPLOAD_TOKEN: ${{ secrets.CODECOV_ORG_TOKEN }}
```

Comment:
Is our intent to test our githubOIDCToken auth functionality for the oidc package example by using:
```
permissions:
  id-token: write
```
from above ?

If so, we can omit this env var `OIDC_UPLOAD_TOKEN`, which is only needed for orgToken auth and imo a bit confusing if it's also present here.

I guess we'll need to do 2 steps of releases. 1) release the changes for the rollup-plugin to support the new auth option and 2) release the oidc example using the upgraded rollup-plugin